Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +36,51 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
```

Comment:
Try to split this method into parts. For example into parts where you add top, bottom, left and right frames, where you determine the count of dashes, where you populate the frame with a message.